src/sentry/search/utils.py

```diff
@@ -145,6 +147,26 @@ def get_date_params(value, from_field, to_field):
         })
     return result
 
+reserved_tag_names = frozenset([
```

Comment:
For what it's worth, we generally do not indent this way but either up to the opening paren or four spaces from the start.